res/stylesheet-win-linux.qss

```diff
@@ -107,10 +107,16 @@ QPushButton#SUV2ROI_Button {
 }
 
 QPushButton:disabled, QPushButton[QPushButtonClass="success-button"]:disabled,
+QPushButton[QPushButtonClass="draw-roi-button"]:disabled,
+QPushButton[QPushButtonClass="reset-button"]:disabled,
 QPushButton[QPushButtonClass="fail-button"]:disabled {
     background-color: #b3b3b3;
 }
 
+QPushButton[QPushButtonClass="draw-roi-button"] {
+    min-height: 30px;
```

Comment:
did we get feedback if we could use the 30px? I think it looks good though

Reply:
Still waiting for Helena's response. But, It only applies the 30px on the Draw ROI interface buttons. Does not affect other buttons.

Reply:
I just hassled Helena.

Reply:
I had a deep investigation on the stylesheet yesterday, It's not just about the QPushbutton size, the QLineEdit min-height is 36px as well. I think this is one of the reasons. If we gonna change the QPushbutton min-height straight away, we have to change the QLineEdit as well(it's gonna involve more stuff). So the safe and easy way is only to apply these changes on Draw ROI UI.